src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FigureSupplier.FIGURE_COUNT];
+        FigureSupplier fs = new FigureSupplier();
```

Comment:
```suggestion
        FigureSupplier figureSupplier = new FigureSupplier();
```

informative variable names